src/ensembl/src/content/app/AppBar.scss

```diff
@@ -1,57 +0,0 @@
-@import 'src/styles/common';
-
-.appBar {
-  font-size: 12px;
-  padding: 8px 20px;
-  min-height: 76px;
```

Comment:
Note that with removal of this component we are losing the minimum height that Andrea wanted this section to have. Need to make sure that all the individial app bars have this minimal height.